src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -313,6 +313,9 @@ extern const char* resume_vr_grammars;
 
 extern const char* ign_off_count;
 
+extern const char* global_ign_on_counter;
+extern const char* global_ign_off_counter;
```

Comment:
do we still need this counter?

Reply:
@yang1070 Removed in bf125f95f4e65b240d46a7a01c8e9bd1a134b983